src/main.ts

```diff
@@ -1595,18 +1607,18 @@ export default class WhatsApp {
     return async (e: WebSocket.OpenEvent) => {
       if (
         !restoreSession ||
-        (restoreSession && !(await doesFileExist(this.keysPath!)))
+        (restoreSession && !(await doesFileExist(this.parameters.keysPath!)))
```

Comment:
```suggestion
        (restoreSession && !(await doesFileExist(this.opts.keysPath!)))
```
Update property again